docker/docker-compose.yml

```diff
@@ -76,6 +76,37 @@ services:
       # needed this volume mapping so changes to partials would be reflected in running app.
       - ../src/angular-app:/var/www/src/angular-app
 
+  lfmerge:
+    image: ghcr.io/sillsdev/lfmerge:${LFMERGE_VERSION:-latest}
+    container_name: lfmerge
+    platform: linux/amd64
+    depends_on:
+      - db
+      - ld-api
+    environment:
+      - WAIT_HOSTS=db:27017
+      - ENVIRONMENT=development
+      - DATABASE=scriptureforge
+      - MONGODB_CONN=mongodb://db:27017
+      - LANGUAGE_DEPOT_API_TOKEN=bogus-development-token
+      - LFMERGE_LOGGING_DEST=syslog
+      - LFMERGE_BASE_DIR=/var/lib/languageforge/lexicon/sendreceive
+      - LFMERGE_WEBWORK_DIR=webwork
+      - LFMERGE_TEMPLATES_DIR=Templates
+      - LFMERGE_MONGO_HOSTNAME=db
+      - LFMERGE_MONGO_PORT=27017
+      - LFMERGE_MONGO_MAIN_DB_NAME=scriptureforge
+      - LFMERGE_MONGO_DB_NAME_PREFIX=sf_
+      - LFMERGE_VERBOSE_PROGRESS=true
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
remind me why this is required?

Reply:
I don't remember right now. It had something to do with being able to access `localhost` from inside the container and have it redirected to the Mongo database mounted on a port of the host machine, but I'll have to look up why I needed that when I get back to the office.

Reply:
Correct on the localhost thing.  I don't think it's for mongo though.  We have it commented out in the LF app and only use it for xdebug (which needs to access localhost)

Reply:
Maybe you can remove it and it will still work?

Reply:
Let's try to remember why and if it is still needed, let's capture that "why" in a comment.

Reply:
It's used in xdebug (see docker/app/docker-php-ext-xdebug.ini), so it's needed in the app and app-for-playwright containers. But it's not needed in the lfmerge container. Will remove.

Reply:
Done in commit 6e74b41.